monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt-br.md

```diff
@@ -2,77 +2,77 @@
 
 Monica Mizzi | 12/09/18 Entrevista com @sunchakr do Monero Talk.
 
-Discussões e debates sobre blockchain são freqüentemente focados na dissecação de aspectos técnicos da tecnologia revolucionária, mas ** existe um tópico não técnico que nunca deixa de chamar a atenção: mulheres em blockchain. **
+Discussões e debates sobre blockchain são frequentemente focados na análise dos aspectos técnicos da tecnologia revolucionária, mas **existe um tópico não técnico que nunca deixa de chamar a atenção: mulheres em blockchain.**
 
-É um tópico que provou atrair elogios e zombarias de mulheres e homens. Embora seja difícil negar o desequilíbrio entre os sexos na indústria, com [falta de estatísticas] (https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality) Destacando o menor número de mulheres investindo e trabalhando na indústria, as opiniões das pessoas sobre o assunto permanecem muito diferentes.
+É um tópico que atrai tanto elogios quanto zombarias igualmente de mulheres e homens. Embora seja difícil negar o desequilíbrio entre os sexos na indústria, o que [não carece de estatísticas] (https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality) que destaquem o menor número de mulheres investindo e trabalhando na indústria, as opiniões das pessoas sobre o assunto ainda são muito diferentes.
```

Comment:
Hey leonardo, 

Remove the spaces between links [não carece de estatísticas] (https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality), so the link won't show, instead only the phrase its connected to. 

[não carece de estatísticas](https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality)